### PR TITLE
Add [sig-autoscaling] tag to remaining cluster autoscaler tests

### PR DESCRIPTION
**Pull request by wojtek-t**
Ref #98326 

```release-note
NONE
```

/kind cleanup
/priority important-soon

### CODE REVIEW

File: test/e2e/autoscaling/cluster_autoscaler_scalability.go [hunk 1]
```diff
@@ -59,7 +59,7 @@ type scaleUpTestConfig struct {
 	expectedResult *clusterPredicates
 }
 
-var _ = framework.KubeDescribe("Cluster size autoscaler scalability [Slow]", func() {
+var _ = SIGDescribe("Cluster size autoscaler scalability [Slow]", func() {
```
Comment (aojea) [inline test/e2e/autoscaling/cluster_autoscaler_scalability.go:62]: is `"[sig-autoscaling] ` the right label?

Comment (aojea) [reply test/e2e/autoscaling/cluster_autoscaler_scalability.go:62]: https://github.com/kubernetes/kubernetes/blob/18605d88141df048af86650e18a4cc7aa5cd0b23/test/e2e/autoscaling/framework.go#L23

Comment (wojtek-t) [reply test/e2e/autoscaling/cluster_autoscaler_scalability.go:62]: it is

Comment (aojea) [reply test/e2e/autoscaling/cluster_autoscaler_scalability.go:62]: nevermind, you added it in the commit message

**Pull request merged**